### ISSUE TITLE
MountVolume.SetUp failed for volume "cloud-config" : secret "proxmox-csi-plugin" not found 

### ISSUE THREAD

**Issue by outbackdingo** [sergelogvinov/proxmox-csi-plugin#279]
i followed the install guide, and using talos, so it seems a step is missing

Install the plugin by using Talos machine config
If you're running [Talos](https://www.talos.dev/) you can install Proxmox CSI plugin using the machine config

```cluster:
  externalCloudProvider:
    enabled: true
    manifests:
      - https://raw.githubusercontent.com/sergelogvinov/proxmox-csi-plugin/main/docs/deploy/proxmox-csi-plugin.yml
```

im getting a Warning  FailedMount  1s (x8 over 64s)    kubelet            MountVolume.SetUp failed for volume "cloud-config" : secret "proxmox-csi-plugin" not found

**Comment by sergelogvinov** [comment]
Hello https://github.com/sergelogvinov/proxmox-csi-plugin/blob/main/docs/install.md#install-the-plugin-by-using-talos-machine-config You should definitely take a look at it 👍

**Comment by outbackdingo** [comment]
yes i saw that and configured as suck is described, so  i re-read the document and noticed this line..

Upload the configuration to the Kubernetes as a secret

kubectl -n csi-proxmox create secret generic proxmox-csi-plugin --from-file=config.yaml

i tried it and it finally has come up and running, though now ive got 

```Warning  FailedScheduling  25m (x16 over 81m)  default-scheduler  0/6 nodes are available: pod has unbound immediate PersistentVolumeClaims. preemption: 0/6 nodes are avail │
│ able: 6 Preemption is not helpful for scheduling.
```
```kubectl get pvc -A
NAMESPACE     NAME                             STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   VOLUMEATTRIBUTESCLASS   AGE
vaultwarden   vaultwarden-data-vaultwarden-0   Pending 
```                                                    <unset>                 118m
```
➜  proxmox git:(main) ✗ kubectl get sc
NAME               PROVISIONER                RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
proxmox-data       csi.proxmox.sinextra.dev   Delete          WaitForFirstConsumer   true                   3h10m
proxmox-data-xfs   csi.proxmox.sinextra.dev   Delete          WaitForFirstConsumer   true                   3h10m```
